src/content/en/2019/third-parties.md

```diff
@@ -163,9 +168,12 @@ The interest of site owners in understanding and analyzing user behavior is not
 
 The top-heavy concentration of script execution is great for the potential impact of performance improvements, but less exciting for the privacy ramifications. 29% of _all_ script execution time across the web is just from scripts on domains owned by Google or Facebook. That’s a very large percentage of CPU time that is controlled by just two entities. It’s critical to ensure that the same privacy protections held to analytics providers be applied in these other ad, social, and developer utility categories as well.
 
-## Caveats
+## Conclusion
 
-- All data presented here is based on a non-interactive, cold load. These values could start to look quite different after user interaction.
-- Third-party content served from a first-party domain is counted as first-party content. i.e. self-hosting Google Fonts or bootstrap.css will be considered first-party content.
-- First-party content served from a third-party domain is counted as third-party content. i.e. first-party images served over a third-party CDN will be considered third-party content.
-- Roughly 84% of all third-party domains by request volume have been identified and categorized. The remaining 16% fall into the “Other” category.
+This chapter has shown that use of third-party content is rampant. This is entirely unsurprising - the entire basis of the web is to allow interconnectiveness and linking.
+
+[Reuse in computer technologies](https://en.wikipedia.org/wiki/Code_reuse) is generally a good thing, though it does introduce depencies which can impact performance (in positive and negative ways!) and has privacy implications, both of which we have touched on here. We have also not discussed the security implications of adding such dependencies to your site, though the security chapter has a [section on vulnerable JS libraries](./security#vulnerable-js-libraries) which is relevant and may be of interest to readers. Even if code has no security concerns it can introduce a single point of failure [which has been recognised as a potenital problem for some time now](https://www.stevesouders.com/blog/2010/06/01/frontend-spof/).
```

Comment:
```suggestion
[Reuse in computer technologies](https://en.wikipedia.org/wiki/Code_reuse) is generally a good thing, though it does introduce dependencies which can impact performance (in positive and negative ways!) and has privacy implications, both of which we have touched on here. We have also not discussed the security implications of adding such dependencies to your site, though the security chapter has a [section on vulnerable JS libraries](./security#vulnerable-js-libraries) which is relevant and may be of interest to readers. Even if code has no security concerns it can introduce a single point of failure [which has been recognized as a potential problem for some time now](https://www.stevesouders.com/blog/2010/06/01/frontend-spof/).
```

Reply:
two typos, but otherwise looks great thank you for doing this! I may make a few tweaks when I go back for the domain/CDN explanation